src/app/core/tasks/marathon/mailserver/controller.nim

```diff
@@ -35,4 +43,164 @@ QtObject:
+      error "error: ", methodName="mailserver-getCurrentMailservers", errDesription
+
+  proc getFleet*(self: MailserverController): string =
+    let settings = self.getCurrentSettings()
+    var fleet = settings_service.DEFAULT_FLEET
+    if(settings.fleet.len > 0):
+      fleet = settings.fleet
+    return fleet
+
+  proc getWakuVersion*(self: MailserverController): int =
+    let nodeConfiguration = self.getCurrentNodeConfiguration()
+    if nodeConfiguration.WakuConfig.Enabled:
+      return WAKU_VERSION_1
+    elif nodeConfiguration.WakuV2Config.Enabled:
+      return WAKU_VERSION_2
+    
+    error "error: unsupported waku version", methodName="mailserver-getWakuVersion"
+    return 0
+
+  proc getPinnedMailserver*(self: MailserverController): string =
+    let settings = self.getCurrentSettings()
+    let fleet = self.getFleet()
+
+    if (fleet == $Fleet.Prod):
+      return settings.pinnedMailserver.ethProd
+    elif (fleet == $Fleet.Staging):
+      return settings.pinnedMailserver.ethStaging
+    elif (fleet == $Fleet.Test):
+      return settings.pinnedMailserver.ethTest
+    elif (fleet == $Fleet.WakuV2Prod):
+      return settings.pinnedMailserver.wakuv2Prod
+    elif (fleet == $Fleet.WakuV2Test):
+      return settings.pinnedMailserver.wakuv2Test
+    elif (fleet == $Fleet.GoWakuTest):
+      return settings.pinnedMailserver.goWakuTest
+    return ""
+
+  proc dialPeer*(self: MailserverController, address: string): bool =
+    try:
+      let response = status_general.dialPeer(address)
+      if response.result.hasKey("error"):
+        let errMsg = $response.result
+        error "waku peer could not be dialed", methodName="mailserver-dialPeer", errMsg
+        return false
+      return true
+    except Exception as e:
+      let errDesription = e.msg
+      error "error: ", methodName="mailserver-dialPeer", errDesription
+      return false
+
+  proc generateSymKeyFromPassword*(self: MailserverController): string =
+    try:
+      let response = status_general.generateSymKeyFromPassword(STATUS_MAILSERVER_PASS)
+      let resultAsString = $response.result
+      return resultAsString.strip(chars = {'"'})
+    except Exception as e:
+      let errDesription = e.msg
+      error "error: ", methodName="mailserver-dialPeer", errDesription
+
+  proc setMailserver*(self: MailserverController, peer: string) =
+    try:
+      let response = status_mailservers.setMailserver(peer)
+    except Exception as e:
+      let errDesription = e.msg
+      error "error: ", methodName="mailserver-setMailserver", errDesription
+
+  proc update*(self: MailserverController, peer: string) =
+    try:
+      let response = status_mailservers.update(peer)
+    except Exception as e:
+      let errDesription = e.msg
+      error "error: ", methodName="mailserver-update", errDesription
+
+  proc requestAllHistoricMessages*(self: MailserverController) =
+    try:
+      let response = status_mailservers.requestAllHistoricMessages()
```

Comment:
It seems like you never use the responses. You can discard instead

Reply:
@jrainville I printed out responses and that's why I left it, but yes, you have right, I removed all such occurrences.